src/storm-pomdp/api/verification.h

```diff
@@ -43,16 +47,20 @@ typename storm::pomdp::modelchecker::BeliefExplorationPomdpModelChecker<storm::m
  * @return the result structure
  */
 template<typename ValueType>
-typename storm::pomdp::modelchecker::BeliefExplorationPomdpModelChecker<storm::models::sparse::Pomdp<ValueType>>::Result underapproximateWithoutHeuristicValues(
-    storm::Environment const& env, std::shared_ptr<storm::models::sparse::Pomdp<ValueType>> pomdp,
-    storm::modelchecker::CheckTask<storm::logic::Formula, ValueType> const& task, uint64_t sizeThreshold,
-    std::vector<std::vector<std::unordered_map<uint64_t, ValueType>>> pomdpStateValues) {
+typename storm::pomdp::modelchecker::BeliefExplorationPomdpModelChecker<
+    storm::models::sparse::Pomdp<ValueType, storm::models::sparse::StandardRewardModel<ValueType>>>::Result
+underapproximateWithoutHeuristicValues(storm::Environment const& env,
+                                       std::shared_ptr<storm::models::sparse::Pomdp<ValueType, storm::models::sparse::StandardRewardModel<ValueType>>> pomdp,
+                                       storm::modelchecker::CheckTask<storm::logic::Formula, ValueType> const& task, uint64_t sizeThreshold,
+                                       std::vector<std::vector<std::unordered_map<uint64_t, ValueType>>> pomdpStateValues) {
     storm::pomdp::modelchecker::BeliefExplorationPomdpModelCheckerOptions<ValueType> options(false, true);
     options.skipHeuristicSchedulers = true;
     options.useClipping = false;
     options.useStateEliminationCutoff = false;
     options.sizeThresholdInit = sizeThreshold;
-    storm::pomdp::modelchecker::BeliefExplorationPomdpModelChecker<storm::models::sparse::Pomdp<ValueType>> modelchecker(pomdp, options);
+    storm::pomdp::modelchecker::BeliefExplorationPomdpModelChecker<
```

Comment:
I dont understand why you need this change... it makes the code harder to read?

Reply:
Seems like a combination of a change that was not really necessary and some auto-formatting my editor did while committing apparently. I reverted the changes and everything still seemed to work, so let's try that.